staging/src/k8s.io/kms/scripts/connect-registry.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
I think this script is better suited to be in https://github.com/kubernetes/kubernetes/tree/master/test/e2e/testing-manifests/auth/encrypt. This is only required for the CI.